asllib/doc/ASLTypingReference.tex

```diff
@@ -730,30 +1095,30 @@ \subsection{Prose}
 
   \item All of the following apply:
     \begin{itemize}
-    \item \texttt{S} has the structure of a string type;
-    \item \texttt{T} has the structure of a string type.
+    \item \texttt{s} has the underlying type of a string type;
+    \item \texttt{t} has the underlying type of a string type.
     \end{itemize}
 
   \item All of the following apply:
     \begin{itemize}
-    \item \texttt{S} has the structure of a boolean type;
-    \item \texttt{T} has the structure of a boolean type.
+    \item \texttt{s} has the underlying type of a boolean type;
+    \item \texttt{t} has the underlying type of a boolean type.
     \end{itemize}
 
   \item All of the following apply:
     \begin{itemize}
     \item \texttt{S} has the structure of an enumeration type;
```

Comment:
should be underlying type instead of structure and lower case s and t

Reply:
done